docs/faq/general.txt

```diff
@@ -32,8 +32,9 @@ thrilled to be able to give something back to the open-source community.
 What does "Django" mean, and how do you pronounce it?
 =====================================================
 
-Django is named after `Django Reinhardt`_, a jazz manouche guitarist from the 1930s
-to early 1950s. To this day, he's considered one of the best guitarists of all time.
+Django is named after `Django Reinhardt`_, a jazz manouche guitarist from the
+1930s to early 1950s. To this day, he's considered one of the best guitarists of
+all time.
```

Comment:
This looks like a lot of churn for minimal benefit.
Can the tool be configured to allow a few more characters per line?

Reply:
@merwok Thank you for taking the time to take a look. At this time, we are not looking at changing the well established Django docs style: we want to **solely focus** on automating it. Therefore, we won't be changing line length in this work.

Reply:
Would you consider making one commit to add the tool and one to change many files, so that the second one can be ignored by git history? (see https://www.michaelheap.com/git-ignore-rev/)